spec/sms_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../lib/sms'
+require 'time'
+
+describe SMS do
```

Comment:
I'd recommend checking out https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#stubbing-the-twilio-api (which I also linked in your cohort channel earlier today) as inspiration of how you can test the SMS class without actually sending the messages